lib/vamble-core/src/vamble_core/event_manager.py

```diff
@@ -27,6 +27,25 @@ def get_event(self, event_id: int):
         if event:
             return self.get_event_model(event)
 
+    def create_event(self, name, description, esport_name, parent_event_name, team_names, ranks):
+        '''Creates and returns a new team record.'''
+        new_event = Event()
+        new_event.name = name
+        new_event.description = description
+        # query IDs for esport, parent event
+        new_event.esport_id = self.session.query(ESport).filter_by(name=esport_name).first().id
+        new_event.parent_event_id = self.session.query(Event).filter_by(name=parent_event_name).first().id
+        self.session.add(new_event)
+        for i in range(len(team_names)):
+            team_name, rank = team_names[i], ranks[i]
```

Comment:
don't really love this pattern. Let's give arbitrary ranks to initial teams added to events. So `rank = i` instead of `rank = ranks[i]` and then it removes the need for `ranks` to be passed as a param anyway.